app/views/admin/admin_users/_user_creations_summary.html.erb

```diff
@@ -0,0 +1,18 @@
+<% unless @works.empty? %>
+  <div class="work listbox group" id="works-summary">
+    <h3 class="heading"><%= t(".works", count: works.size) %></h3>
```

Comment:
This `works.size` and the `comments.size` are always the pagination size (20 on prod, 25 in dev) when there is more than one page of works/comments. That is the case even when on a page with less than the page size works/comments.

It looks like using `works.count` would make it show the total count instead, which would be less confusing. Ideally I think it would be nice to list something similar to work search results, so "1 - 20 of 34 Works" etc, but I don't know if that is possible